support/pom.xml

```diff
@@ -34,12 +34,12 @@
         <dependency>
             <groupId>net.sf.saxon</groupId>
             <artifactId>Saxon-HE</artifactId>
-            <version>12.2</version>
+            <version>11.5</version>
         </dependency>
         <dependency>
             <groupId>com.xmlcalabash</groupId>
             <artifactId>xmlcalabash</artifactId>
-            <version>1.5.6-120</version>
+            <version>1.5.3-110</version>
```

Comment:
These appear to be earlier dependency versions than what was used before. Is there a reason for this?

Reply:
Deferring this to @nikitawootten-nist who is helping me understand and maintain dependencies.